Documentation/hubble.rst

```diff
@@ -98,11 +100,22 @@ Cilium's datapath node handler interface.
 Hubble Relay
 ------------
 
-.. note:: At the time of this writing, Hubble Relay component is still
-          work in progress and may undergo major changes. For this reason,
-          internal documentation about Hubble Relay is limited.
-
-Hubble Relay is a component that was introduced in the context of multi-node
-support. It leverages the Peer service to obtain information about Hubble
-instances and consume their gRPC API in order to provide a more rich API that
-covers events from across the entire cluster.
+Hubble Relay is the Hubble component that brings multi-node support. It
+leverages the Peer service to obtain information about Hubble instances and
+consume their gRPC API in order to provide a more rich API that covers events
+from across the entire cluster (or even multiple clusters in a ClusterMesh
+scenario).
+
+Hubble Relay was first introduced as a technology preview with the release of
+Cilium v1.8. It is declared stable with the release of Cilium v1.9.
+
+Hubble Relay implements the Observer service for multi-node. To that end, it
+maintains a persistent connection with every Hubble peer in a cluster with a
+peer manager. This component provides callers with the list of peers. Callers
+may report when a peer is unreachable, in which case the peer manager will
+attempt to reconnect.
+
+As Hubble Relay connects to every node in a cluster, the Hubble server
+instances must make their API available (by default on port 4244). By default,
+Hubble server endpoints are secured using mutual TLS (mTLS) when exposed on a
+TCP port in order to limit access to Hubble Relay only.
```

Comment:
nit:
```suggestion
TCP port in order to authorize access from Hubble Relay only.
```

Reply:
"limit" is more accurate I think as there are two parts: a) authorization b) sending data. "authorize" only implies a), whereas we want both. If mTLS was only used for authorization and then data was sent over a different channel then "authorize" would be correct.